klippy/extras/screws_tilt_adjust.py

```diff
@@ -73,6 +75,11 @@ def probe_finalize(self, offsets, positions):
         else:
             # First screw is the base position used for comparison
             i_base, z_base = 0, positions[0][2]
+
+            if self.adjust_to_z is not None:
```

Comment:
Note that this will only work if `DIRECTION` is not set. This may lead to confusing results: if a user specifies `ADJUST_TO_Z`, they would expect that the recommendation brings bed to the specified level. However, this won't happen if they accidentally or unknowingly also set `DIRECTION`. I'd suggest to do it like
```
  if self.adjust_to_z is not None:
    ...
  elif self.direction is not None:
    ...
  else:
    # First screw is the base position used for comparison
    i_base, z_base = 0, positions[0][2]
```
I'd suggest to mention this behavior in the documentation. And ideally, the code in `cmd_SCREWS_TILT_CALCULATE` should check that both options are not set together (in general, it is impossible to meet both requirements) and report an error.

Reply:
Yes it is intented to work with DIRECTION not set as that would not make sense. Sure i can check if both are set together and output something

Reply:
Well, you should always expect errors in the user input unless validated :) Especially in this potentially non-obvious case.